libs/ts/config-types/src/node-config/types.ts

```diff
@@ -39,15 +39,22 @@ const PublishingCriteriaSchema = S.Struct({
   peg_tolerance_percentage: S.Number,
 });
 
+const ContractSchema = S.Struct({
+  name: S.String,
```

Comment:
Do we know the names of the contracts that can be passed here. If so lets make it string literal